Gemfile

```
@@ -51,6 +51,10 @@ gem "bootsnap", require: false
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
   gem "debug", platforms: %i[ mri mingw x64_mingw ]
+  gem 'rspec-rails', '6.0.1'
+  gem 'shoulda', '~> 4.0'
+  gem "factory_bot", "~> 6.2"
+  gem 'factory_bot_rails'
```

I think you don't actually need to have both factory_bot and factory_bot_rails in the gem file. You can probably remove factory_bot <3

```suggestion
  gem 'factory_bot_rails'
```

---

Sure thing! Thanks!